microsetta_interface/templates/nutrition.jinja2

```diff
@@ -72,13 +72,15 @@
             });
         });
 
+        /*
```

Comment:
should this function be deleted?

Reply:
I'd like to revisit making the button responsive to user input, so I'd prefer not to delete the code entirely.